packages/utils.py

```diff
@@ -488,6 +488,10 @@ def default(self, obj):
             data = {attr: getattr(obj, attr) for attr in self.pkg_attributes}
             for attr in self.pkg_list_attributes:
                 data[attr] = getattr(obj, attr).all()
+            all_deps = obj.depends.all()
+            for (deptype, name) in [('D', 'depends'), ('O', 'optdepends'),
+                                    ('M', 'makedepends'), ('C', 'checkdepends')]:
```

Comment:
Just a coding style thing, I'd use a constant called DEPENDENCIES_TYPE with that list, and iterate that instead